package.json

```diff
@@ -56,7 +56,7 @@
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-1": "6.24.1",
     "chai": "3.5.0",
-    "conventional-changelog": "1.1.3",
+    "conventional-github-releaser": "^1.1.9",
     "css-loader": "0.26.4",
     "del": "2.2.2",
     "documentation": "4.0.0-beta.19",
```

Comment:
I would let this stay at beta.18 as long as you didn't push your documentationjs updates.